tests/kubectl_test.go

```diff
@@ -0,0 +1,67 @@
+package tests_test
+
+import (
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = FDescribe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kubectl get vm/vmi tests", func() {
```

Comment:
Hi @vatsalparekh is the test feature ready? If so can you please remove the F here?

Reply:
removed